python/fbprophet/models.py

```diff
@@ -128,7 +128,7 @@ def sampling(self, stan_init, stan_data, samples, **kwargs) -> dict:
                                           inits=stan_init,
                                           iter_sampling=iter_half,
                                           **kwargs)
-        res = self.stan_fit.sample
+        res = self.stan_fit.draws()
```

Comment:
The `.sample` property now has a deprecation warning, and the guidance is to use `.draws()` instead.